lua/ocaml/init.lua

```diff
@@ -77,4 +77,64 @@ function ocaml.construct(input)
   end)
 end
 
+function ocaml.jump()
+  with_server(function(client)
+    local params = vim.lsp.util.make_position_params(0, client.offset_encoding)
+    local result = client.request_sync("ocamllsp/jump", params, 1000)
+    if not (result and result.result) then
+      vim.notify("Unable to jump.", vim.log.levels.WARN)
+      return
+    end
+    local jumps = result.result.jumps
+    if #jumps == 0 then
+      vim.notify("Nothing to jump to.", vim.log.levels.INFO)
+      return
+    end
+    local answer = vim.fn.input("target: ")
```

Comment:
instead of asking this here, why we not accept it in argument?

It will be faster if I send this command right away `Jump match`. I think we can do that